apps/central-scan/backend/src/importer.ts

```diff
@@ -22,23 +22,19 @@ import { join } from 'path';
 import { v4 as uuid } from 'uuid';
 import { BatchControl, BatchScanner } from './fujitsu_scanner';
 import { Castability, checkSheetCastability } from './util/castability';
-import { HmpbInterpretationError } from './util/hmpb_interpretation_error';
 import { Workspace } from './util/workspace';
 import {
   describeValidationError,
   validateSheetInterpretation,
 } from './validation';
-import * as workers from './workers/combined';
 import * as interpretNhWorker from './workers/interpret_nh';
 import * as interpretVxWorker from './workers/interpret_vx';
```

Comment:
Maybe call these "interpreters" rather than "workers"? That's how we colloquially talk about them